incubator/group/keeper_test.go

```diff
@@ -105,29 +109,29 @@ func TestCreateGroup(t *testing.T) {
 func TestCreateGroupAccount(t *testing.T) {
 	amino := codec.New()
 	pKey, pTKey := sdk.NewKVStoreKey(params.StoreKey), sdk.NewTransientStoreKey(params.TStoreKey)
-	paramSpace := subspace.NewSubspace(amino, pKey, pTKey, DefaultParamspace)
+	paramSpace := subspace.NewSubspace(amino, pKey, pTKey, group.DefaultParamspace)
 
-	groupKey := sdk.NewKVStoreKey(StoreKeyName)
-	k := NewGroupKeeper(groupKey, paramSpace, baseapp.NewRouter(), &MockProposalI{})
-	ctx := NewContext(pKey, pTKey, groupKey)
-	k.setParams(ctx, DefaultParams())
+	groupKey := sdk.NewKVStoreKey(group.StoreKeyName)
+	k := group.NewGroupKeeper(groupKey, paramSpace, baseapp.NewRouter(), &group.MockProposalI{})
+	ctx := group.NewContext(pKey, pTKey, groupKey)
+	defaultParams := group.DefaultParams()
+	paramSpace.SetParamSet(ctx, &defaultParams)
```

Comment:
Same as above

Reply:
This is not possible as tests are in `group_test`